test/resources/nightly-config.yaml

```diff
@@ -0,0 +1,99 @@
+#!productName=RHBA
+#!groovyScriptsPath=https://groovyScriptsPathUrl
+#!datetimeSuffix=
+#!communityVersion=7.33.0
+#!installerCommonsVersion=2.3.6.rhba
+#!productVersion=
+#!izpackVersion=4.5.2.rhba-redhat-00006
+#!erraiVersion=4.7.0.Final
+#!mvelVersion=2.4.3.Final-redhat-1
+#!thorntailVersion=2.5.1.Final-redhat-00010
+
+# Comprehensive list of RHBA products versions updatable with the Update Tool in the current build
+#!updatableVersionRange=7.4.1.GA-redhat-00001,7.5.1.redhat-00001
+
+#!milestone=NIGHTLY
+#!stage=GA
+#!scmRevision=main
+#!projectB-scmRevision={{scmRevision}}
+#!projectC-scmRevision={{scmRevision}}
+
+#!versionOverrideCommunity=-DversionOverride={{communityVersion}}
+#!versionOverrideUberfire=-DversionOverride={{communityVersion}}
+#!versionOverrideProduct=-DversionOverride={{productVersion}}
+#!versionOverrideIC=-DversionOverride={{installerCommonsVersion}}
+#!versionOverrideIzpack=-DversionOverride={{izpackVersion}}
+#!versionSuffix=-DversionSuffix=redhat-{{datetimeSuffix}}
+
+#!gitServer=gitServerUrl
+
+#!npm_registry=https://url/registry.npmjs.org/
+#!node_download=http://url/node/
+#!npm_download=http://url/npm/
+#!yarn_download=http://url/yarn/
+#!cypress_download=http://url/npm-libraries/cypress-3.7.0.zip
+
+
+#!baseMaven=mvn deploy -B -Dfull=true -Drevapi.skip=true -Denforcer.skip=true -Dgwt.compiler.localWorkers=1 -Dproductized=true -Dfindbugs.skip=true -Dcheckstyle.skip=true
+#!baseMavenTree={{baseMaven}}
+#!groovyScriptBase={{groovyScriptsPath}}
+
+#!starrDefine=-Dstarr.version=2.12.8.redhat-00001
+#!rhbaNameSuffix=
```

Comment:
please remove all copy&paste lines which we don't really need in frame of drools ansible integ.